src/components/select-field/BaseSelectField.js

```diff
@@ -208,6 +208,11 @@ class BaseSelectField extends React.Component<Props, State> {
                     this.closeDropdown();
                 }
                 break;
+            case 'Tab':
+                if (isOpen) {
```

Comment:
Do we want the default event to still occur when `tab` is pressed?

Reply:
Yes, the default case stops the default events, causing the keyboard trap. We want the default events (keyboard navigation) when `tab` is pressed. @booc0mtaco